src/Microsoft.DotNet.Darc/tests/Microsoft.DotNet.DarcLib.Tests/VirtualMonoRepo/VmrPatchHandlerTests.cs

```diff
@@ -35,6 +37,12 @@ public class VmrPatchHandlerTests
         "https://github.com/dotnet/external-1",
         SubmoduleSha1);
 
+    private readonly GitSubmoduleInfo _nestedSubmoduleInfo = new(
```

Comment:
If this is used in one test only, I'd just create it locally inside that test. I think the others are re-used between multiple tests.